tests/Tools/EnvironmentUtilityTest.php

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+
+namespace PHPSu\Tests\Tools;
+
+use PHPSu\Exceptions\CommandExecutionException;
+use PHPSu\Tools\EnvironmentUtility;
+use PHPUnit\Framework\TestCase;
+
+class EnvironmentUtilityTest extends TestCase
+{
+    public function testCommandIsInstalled(): void
+    {
+        $this->assertEquals(true, (new EnvironmentUtility())->isCommandInstalled('echo'));
+    }
+
+    public function testCommandIsNotInstalled(): void
+    {
+        $this->assertEquals(false, (new EnvironmentUtility())->isCommandInstalled('reiguheruh'));
+    }
+
+    public function testGetInstalledRsyncVersionOrExpectExceptionIfNotInstalled(): void
+    {
+        if ((new EnvironmentUtility())->isRsyncInstalled()) {
```

Comment:
mark test as risky, as it dose not test the functions in every case?

Reply:
it tests in every case: if installed, i am expecting a version result, if not installed i expect an exception